include/torch-mlir/Dialect/Torch/Utils/Utils.h

```diff
@@ -150,6 +152,13 @@ Type getDefaultAccType(PatternRewriter &rewriter, Type inputType);
 LogicalResult getPermutedType(BaseTensorType inType,
                               SmallVector<int64_t> permuteDims,
                               Type &permutedType);
+// A helper function to get a constant value vector from an int64_t ArrayRef
+SmallVector<Value> getValueVector(OpBuilder &builder, Location loc,
+                                  ArrayRef<int64_t> intVector);
+
+// A helper function to get a list Type from an int64_t ArrayRef
+PrimListConstructOp getList(OpBuilder &builder, Location loc,
+                            ArrayRef<int64_t> intVector);
```

Comment:
Remove these changes, see the comment below.

Reply:
Done.